ChangeContent.cpp

```diff
@@ -0,0 +1,29 @@
+#include "ProcessManager.h"
+using namespace std;
+
+void ProcessManager :: changeContent(long address, string test ){
+    // FILE* mem_file;
+    // char memFileName[1024];
+    char *memFileName = (char*)malloc(50);
+    sprintf(memFileName,"/proc/%lu/mem",pID);
+    int mem_file = open(memFileName, O_RDWR);
+    if(mem_file == -1 ){
+        printf("couldn't open file");
+        exit(1);
+    }
+    char *buf = (char*)malloc(4);
+    lseek(mem_file, address, SEEK_SET);
+    read(mem_file, buf, 4);
```

Comment:
Are you fixing the size of the chunk we need to read to 4 bytes?

Reply:
No, no, I was just trying for int.